lldb/test/Shell/SymbolFile/DWARF/dwo-static-data-member-access.test

```diff
@@ -0,0 +1,23 @@
+# UNSUPPORTED: system-darwin, system-windows
```

Comment:
Just out of curiosity — we are evaluating a static constant. Do we actually need to *run* the binary, or could we run the expression against e.g., an ELF binary on Darwin, too, by just evaluating the expression and not launching at all?

Reply:
That's actually a great question, but I think for now the answer with lldb is "no, expression evaluation doesn't work without a running process" - but I was just commenting on an internal bug where folks were having trouble investigating core dumps with some of our pretty printers that evaluate expressions that happen to be constants that gdb can evaluate on a core/without a running process, but it seems lldb can't do that... 

So while that sounds like a good thing, my understanding is that lldb can't do that at the moment.

Reply:
As @dwblaikie points out, we need a process to run the expression. But if we specify `-flimit-debug-info` I can repro the crash on Darwin too, so we can remove `UNSUPPORTED` here and get the coverage that way.